gpu-environment.yml

```diff
@@ -0,0 +1,35 @@
+name: tybalt-gpu
+channels:
+- conda-forge
+- defaults
+dependencies:
```

Comment:
Is python omitted? It's probably installed as a dependency I assume? Better to explicitly list